packages/fast-components/src/progress/base-progress.ts

```diff
@@ -0,0 +1,18 @@
+import { attr, FastElement } from "@microsoft/fast-element";
+
+export class BaseProgress extends FastElement {
+    @attr
+    public value: number;
+
+    @attr
+    public min: number;
+
+    @attr
+    public max: number;
+
+    @attr
+    public paused: boolean;
```

Comment:
We could set this to use the new boolean attr behavior as well. Currently this will have a bug if paused is the string value "false", for example. This feature isn't merged yet, but should be merging shortly.

Reply:
If we don't want it to be a boolean attr, we can independently leverage the boolean converter to ensure the type is correct when coming from HTML as well.